common/src/main/java/net/william278/husksync/data/DataServerID.java

```diff
@@ -0,0 +1,37 @@
+package net.william278.husksync.data;
+
+import com.google.gson.annotations.SerializedName;
+import org.jetbrains.annotations.NotNull;
+
+public class DataServerID {
```

Comment:
I think this should probably be a `@YamlDocument` saved in the user's config directory where the user has to enter the server ID.